docs/api/rest_api/rest_api_reference/input/ibexa-types.raml

```diff
@@ -1082,16 +1082,53 @@ LocationCreateStruct:
             required: true
             type: integer
 
+LocationListXML:
+    type: LocationRefXML[]
+    xml:
+        name: LocationList
+    properties:
```

Comment:
It's like LocationList node shouldn't have properties…

`Message: Property 'properties' not supported in a RAML 1.0 array node`

```xml
<?xml version="1.0" encoding="UTF-8"?>
<LocationList media-type="application/vnd.ibexa.api.LocationList+xml" href="/api/ibexa/v2/content/objects/1/locations">
 <Location media-type="application/vnd.ibexa.api.Location+xml" href="/api/ibexa/v2/content/locations/1/2/42"/>
</LocationList>
```